.github/ISSUE_TEMPLATE/4_KIT_release_checks.md

```diff
@@ -3,25 +3,31 @@ name: KIT Release Checks
 about: Use this template to track KIT release checks
 title: "[KIT NAME] Release Checks"
 labels: kit
+milestone: "24.05"
 ---
 
 This issue tracks all relevant release checks for KITs.
 
+> [!IMPORTANT]  
+> Select issue type "release_ac" to make the issue visible in the [Release Management Kanban Board](https://github.com/orgs/eclipse-tractusx/projects/26/views/17)
+
 ## Release Info
 
 Please provide information on what you want to be included in the Eclipse Tractus-X release.
 If you are not owner of this issue, please provide the information as comment to the issue.
+Make sure to assign this issue to expert(s) for their approval, as soon as you have finished preparation.
```

Comment:
insert line above: Best practice: have a professional Peer review your prepared content before you submit for expert approvals.